### DISCUSSION TITLE
How to configure the port?

### DISCUSSION THREAD

**Discussion by JC3** [serialport/node-serialport#2642] [Q&A] [answered]
How do you set options like number of stop bits, parity, flow control, etc.? I can't find any constructor parameters mentioned in the docs besides `baudRate`.

Thanks!

**Comment by GazHank** [comment, marked as answer]
They are defined when you open the port per https://serialport.io/docs/api-bindings-cpp#open

As you can see most of the items are optional as they default to OS specific values

**Reply by JC3** [reply]
Thank you!!! I didn't see that part of the docs. It's all clear now!
